apps/app/src/electron/EverythingService.ts

```diff
@@ -129,25 +107,32 @@ type ConvertToServerSide<T> = {
 		: T[K]
 }
 
+function Undoable(target: EverythingService, _key: string, descriptor: PropertyDescriptor) {
```

Comment:
Should we add a description here explaining that the intended usage is as a decorator?